nexus/types/src/deployment/planning_input.rs

```diff
@@ -249,6 +249,9 @@ pub enum SledFilter {
     /// Sleds that are eligible for discretionary services.
     EligibleForDiscretionaryServices,
 
+    /// Sleds whose sled agents should be queried for inventory
+    InInventory,
```

Comment:
I initially misunderstood this value to mean sleds that are in inventory, and not ones that should be queried for it. I'm not sure if that's a reason to rename it to something more specific though. I was thinking something like, `QueryForInventory`

Reply:
Yeah, I hear that.  I went back and forth on a few names and didn't like any of them.

- `ForInventory`, as in "this is the _filter_ used _for_ collecting inventory" but that felt weird -- the other names are not phrases that get attached to the phrase "sled filter"
- `QueryForInventory` made me a little worried someone might think these sleds are not in inventory at all since they're not being queried (but they can be in inventory if MGS can see them)
- `InInventory` (what I have) -- what you said
- `IncludedInInventory` -- same problem, it's misleading
- `QuerySledAgentDurringInventory` -- accurate, but a mouthful

I'm open to these or other ideas but I don't feel especially great about any of these.

Reply:
I'm fine with `InInventory` personally, but what about `QueryDuringInventory`?

Reply:
I think that's about the same as `QueryForInventory`.  The confusion I'm worried about there is: you have to know how inventory works (namely, that we also query MGS) to understand that even if something is not queried, it may appear in the inventory.  But this isn't a big deal.  And `InInventory` isn't really better.  I'll go ahead and use `QueryForInventory` for now.